example/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "dev": "astro dev",
     "dev:single-entrypoints": "astro dev --config astro.config.ts",
+    "dev:module": "astro dev --config astro.module.config.ts",
```

Comment:
```suggestion
```

If we remove the `astro.module.config.ts` example, we can also remove this script.